app/services/scenes/scene.ts

```diff
@@ -465,8 +466,6 @@ export class Scene {
       if (nodeModel.sceneNodeType !== 'folder') return;
       this.getSelection(nodeModel.childrenIds).moveTo(this.id, nodeModel.id);
     });
-
-    this.scenesService.sourcesAdded.next(this.id);
```

Comment:
not sure why this was removed? disregard if it's a subscription that was only used by one thing that's not necessary anymore

Reply:
Since we now assign the displays when when loading the `scene-items` node, and assign the contexts when loading the item attributes in the `scene-item` service, this subscription was no longer needed and was only used by the dual output service to assign contexts after loading the scene items.